docs/.sphinx/_toc.yml.in

```diff
@@ -12,11 +12,17 @@ root: index.md
 subtrees:
   - caption: User Guide
     entries:
-      - file: user_guide/projects_yaml
+      - file: user_guide/spellcheck
+      - file: user_guide/linting
       - file: user_guide/article_info
   - caption: Developer Guide
     entries:
+      - file: developer_guide/doxygen_integration
```

Comment:
This should be in the user guide. The user guide is for **users** of `rocm-docs-core` (i.e. developers working on the documentation of other projects that make use of `rocm-docs-core`), while the developer guide is for developers of `rocm-docs-core` itself.